pkgs/development/python-modules/easy-thumbnails/default.nix

```diff
@@ -7,37 +7,55 @@
   pythonOlder,
   reportlab,
   svglib,
+  pytestCheckHook,
+  pytest-django,
+  setuptools,
+  testfixtures,
 }:
 
 buildPythonPackage rec {
   pname = "easy-thumbnails";
   version = "2.10";
-  format = "setuptools";
+  pyproject = true;
 
-  disabled = pythonOlder "3.6";
+  disabled = pythonOlder "3.9";
 
   src = fetchPypi {
-    inherit pname version;
+    inherit version;
+    pname = "easy_thumbnails";
     hash = "sha256-0AlGL815Ht8kluXtRtBEcS4aBrfUYA5M8oEgAumQTvU=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     django
     pillow
-    svglib
     reportlab
+    svglib
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-django
   ];
 
-  # Tests require a Django instance which is setup
-  doCheck = false;
+  checkInputs = [ testfixtures ];
+
+  env.DJANGO_SETTINGS_MODULE = "easy_thumbnails.tests.settings";
```

Comment:
It is preferable to do this in `preCheck`, as for some Django projects this affects the build.